src/pages/Engineering.pug

```diff
@@ -70,6 +68,23 @@ block article
 
         # Preventing secondary structures
 
+        Due to the formation of duplex secondary structures, especially for guanine, elongation by TDT is enormously complicated. 
+        This problem often prevents the extension of all strands to more than 1000 nt. 
```

Comment:
For storing longer pieces of information in DNA it is essential to be able to synthesize longer parts.
Therefore, we had to think of methods to avoid the DNA forming complex structures.